docs/tutorials/add-contracts-pallet.md

```diff
@@ -61,15 +61,14 @@ check out [their official documentation](https://doc.rust-lang.org/cargo/referen
 
 Open `substrate-node-template/runtime/Cargo.toml` and you will see a list of all the dependencies
 your runtime has. For example, it depends on the
-[Balances pallet](https://substrate.dev/rustdocs/v2.0.0-rc6):
+[Balances pallet](https://github.com/paritytech/substrate/tree/v2.0.0-rc6/frame/balances):
```

Comment:
Could also reference the docs for this, but not sure where this would be...?

Reply:
https://substrate.dev/rustdocs/v2.0.0-rc6/pallet_balances/

Reply:
```suggestion
[Balances pallet](https://substrate.dev/rustdocs/v2.0.0-rc6/pallet_balances/):
```
Nice to see you here @NukeManDan :wave: